readme.md

```diff
@@ -1015,6 +1015,18 @@ If you liked Cloud Atlas this is a good read.
 
 _Novels which focus on the near-future unintended consequences of biotechnology revolutions._
 
+#### [Pandemic](https://www.goodreads.com/book/show/34811896-pandemic) (2017) *by [A.G. Riddle](https://www.goodreads.com/author/show/3087199.A_G_Riddle)
+
+The first book in a two part series called the extinction files, the story explores a bioterrorist plot attempting to usurp the major world governments, with an extra scifi twist. [@Rogue-System](https://github.com/Rogue-System)
```

Comment:
What is your opinion about this book and series? 

Also, add a dash between the period and your name.